src/App.js

```diff
@@ -19,4 +19,5 @@ function App() {
 }
 
 export default App;
-//yolo fixed it
\ No newline at end of file
+//yolo fixed it
+// made a branch for notes
```

Comment:
Nice job Josh! Within our Main.js component, we typically want to pass props as name={item.name} says={item.says} type={item.type} left={item.left} top={item.top}. Then in Animal.js, we can pass (props) as the argument and utilize dot notation to render elements, such as name === props.name. We will learn destructuring soon, I just want to be sure you fully understand passing down props prior to destructuring.